terraform-tests/mssql_test.go

```diff
@@ -261,4 +265,50 @@ var _ = Describe("mssql", Label("mssql-terraform"), Ordered, func() {
 			})
 		})
 	})
+
+	Context("kms_key_id", func() {
+		When("with default values", func() {
+			It("should succeed", func() {
+				plan = ShowPlan(terraformProvisionDir, buildVars(defaultVars, requiredVars, nil))
+				Expect(ResourceCreationForType(plan, "aws_db_subnet_group")).To(HaveLen(1))
+				Expect(AfterValuesForType(plan, "aws_db_instance")).To(MatchKeys(IgnoreExtras, Keys{"storage_encrypted": BeTrue()}))
+			})
+		})
+
+		When("no kms_key_id passed and storage_encrypted set to true", func() {
+			// Currently, this tests the exact same inputs as the one `with default values`
```

Comment:
suggestion: Interesting take, maybe we can discuss in tech forum and see if we want to make this the standard.